profile/src/test/java/com/scottlogic/deg/profile/reader/JsonProfileReaderTests.java

```diff
@@ -880,21 +880,6 @@ public void nullable_DoesNotAddConstraintForField_whenSetToTrue() throws IOExcep
         expectRules(ruleWithDescription("type-rules"));
     }
 
-    @Test
-    public void nullable_DoesNotAddConstraintForField_whenNotSet() throws IOException  {
-        givenJson(
-            "{" +
-                "    \"schemaVersion\": " + schemaVersion + "," +
-                "    \"fields\": [ { " +
-                "       \"name\": \"foo\"," +
-                "       \"type\": \"integer\"" +
-                "    } ]," +
-                "    \"rules\": []" +
-                "}");
-
-        expectRules(ruleWithDescription("type-rules"));
-    }
-
```

Comment:
I've removed this test.  I spent a fair bit of time trying to update it and I didn't really understand how to write it.  I'm not convinced this test (or the other tests in this file) are adding much value, but I'm open to being convinced otherwise if they've proved useful in the past.

Reply:
i can give you a hand with this tomorrow